### ISSUE TITLE
Instruction to "Open Hyper-V Manager" is ambiguous

### ISSUE THREAD

**Issue by olofh-msft** [MicrosoftDocs/windowsserverdocs#1933]
I do not know how to "Open Hyper-V Manager'.  I see that the hyper-V is listed as 'Installed' in Control Panel-&gt;Programs-&gt;Programs and Features-&gt;Turn Windows features on and off-&gt;Add roles and features wizard-&gt;server roles.   And yet, Searching for "Hyper-V in windows search shows nothing, and there is nothing hyper-V related under windows administrative tools, 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: acb64720-e61b-a57c-bdf1-ccf0fde04e56
* Version Independent ID: dcabb586-ddd7-0041-a0d2-608ceaf2c398
* Content: [Create a virtual switch for Hyper-V virtual machines](https://docs.microsoft.com/en-us/windows-server/virtualization/hyper-v/get-started/create-a-virtual-switch-for-hyper-v-virtual-machines)
* Content Source: [WindowsServerDocs/virtualization/hyper-v/get-started/Create-a-virtual-switch-for-Hyper-V-virtual-machines.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/virtualization/hyper-v/get-started/Create-a-virtual-switch-for-Hyper-V-virtual-machines.md)
* Service: **na**
* Product: **windows-server-threshold**
* GitHub Login: @KBDAzure
* Microsoft Alias: **kathydav**

**Comment by Justinha** [comment]
@olofh-msft In Turn Windows Features on or off, can expand the Hyper-V node and make sure all of the checkboxes below it are selected? Esp Hyper-V GUI Management Tools.

**Comment by olofh-msft** [comment]
Hyper-V GUI Management Tools. Does not appear under the Hyper-V Node.  Here's what I see:

[cid:image001.png@01D47CFC.7E5B6A30]

- Olof

**Comment by Justinha** [comment]
@olofh-msft Thanks for the image. Which version of Windows? Is this Windows Server? In that case you might need to install it using Server manager: https://docs.microsoft.com/en-us/windows-server/virtualization/hyper-v/get-started/install-the-hyper-v-role-on-windows-server#BKMK_SERV

**Comment by olofh-msft** [comment]
Yes, this is Windows Server 2016.  I will follow the link you sent.

**Comment by doctordns** [comment]
This is a duplicate of #1935 

I( have added a PR to clarify that if you don't see the tools you need to load them. #2135

**Comment by IngridAtMicrosoft** [comment]
#please-close
